libs/plugins/stream-client/src/lib/run-stream/run-stream-button/action-status.component.ts

```diff
@@ -17,6 +17,7 @@ export class ActionStatusComponent {
   readonly statuses = StreamProcessStatus;
   @Input() status: StreamProcessStatus;
   @Input() disabled = false;
```

Comment:
I think we need to clarify the difference between `disabled` and `disableRunStream`